wagn/lib/decko/engine.rb

```diff
@@ -12,27 +12,39 @@
 require 'diff/lcs'
 require 'diffy'
 
+require 'wagn'
 
 module Decko
+
   class Engine < ::Rails::Engine
 
     paths.add "app/controllers", :with => 'rails/controllers', :eager_load => true
     paths.add 'gem-assets',      :with => 'rails/assets'
     paths.add 'config/routes',   :with => 'rails/engine-routes.rb'
-    paths.add 'lib/tasks',       :with => 'lib/wagn/tasks', :glob => '**/*.rake'
-    
+    paths.add 'lib/tasks',       :with => "#{::Wagn.gem_root}/lib/wagn/tasks", :glob => '**/*.rake'
+    paths.add 'lib/wagn/config/initializers',
+              :with => File.join( Wagn.gem_root, 'lib/wagn/config/initializers' ), :glob => "**/*.rb"
+
+    initializer 'decko.engine.load_config_initializers',  :after => :load_config_initializers do
+      paths['lib/wagn/config/initializers'].existent.sort.each do |initializer|
+        load(initializer)
+      end
+    end
```

Comment:
are you sure engine decks will want all the application initializers?

It's very hard to get enthusiastic about merging this if none of it has been tested in an engine context.